File: mathvar.cpp
// datavar.cpp
#include <iostream>

int main() {
    int a = 5;
    int b = 3;
    int sum = a + b;
    int diff = a - b;
    int prod = a * b;
    int quot = a / b;

    std::cout << "5 + 3 = " << sum << std::endl;
    std::cout << "5 - 3 = " << diff << std::endl;
    std::cout << "5 * 3 = " << prod << std::endl;
    std::cout << "5 / 3 = " << quot << std::endl;

    return 0;
}

File: mathvar.ml
(* datavar.ml *)
let a = 5
let b = 3
let sum = a + b
let diff = a - b
let prod = a * b
let quot = a / b

let () =
  Printf.printf "5 + 3 = %d\n" sum;
  Printf.printf "5 - 3 = %d\n" diff;
  Printf.printf "5 * 3 = %d\n" prod;
  Printf.printf "5 / 3 = %d\n" quot
